tests/python/rest_api/test_jobs.py

```diff
@@ -1321,47 +1327,82 @@ def _check_cvat_for_video_job_annotations(content, values_to_be_checked):
+    def test_can_export_dataset(self, admin_user: str, jobs_with_shapes: List, api_version: int):
         job = jobs_with_shapes[0]
-        response = self._export_dataset(admin_user, job["id"])
-        assert response.data
+        self._test_export_dataset(
+            admin_user,
+            job["id"],
+            api_version=api_version,
+            local_download=self.is_local_download(job),
+        )
 
-    def test_non_admin_can_export_dataset(self, users, tasks, jobs_with_shapes):
-        job_id, username = next(
+    @pytest.mark.parametrize("api_version", (1, 2))
+    def test_non_admin_can_export_dataset(self, users, jobs_with_shapes, api_version: int):
+        job, username = next(
             (
-                (job["id"], tasks[job["task_id"]]["owner"]["username"])
+                (job, self.tasks[job["task_id"]]["owner"]["username"])
                 for job in jobs_with_shapes
-                if "admin" not in users[tasks[job["task_id"]]["owner"]["id"]]["groups"]
-                and tasks[job["task_id"]]["target_storage"] is None
-                and tasks[job["task_id"]]["organization"] is None
+                if "admin" not in users[self.tasks[job["task_id"]]["owner"]["id"]]["groups"]
+                and self.tasks[job["task_id"]]["target_storage"] is None
+                and self.tasks[job["task_id"]]["organization"] is None
             )
         )
-        response = self._export_dataset(username, job_id)
-        assert response.data
+        self._test_export_dataset(
+            username, job["id"], api_version=api_version, local_download=self.is_local_download(job)
+        )
 
-    def test_non_admin_can_export_annotations(self, users, tasks, jobs_with_shapes):
-        job_id, username = next(
+    @pytest.mark.parametrize("api_version", (1, 2))
+    def test_non_admin_can_export_annotations(self, users, jobs_with_shapes, api_version: int):
+        job, username = next(
             (
-                (job["id"], tasks[job["task_id"]]["owner"]["username"])
+                (job, self.tasks[job["task_id"]]["owner"]["username"])
                 for job in jobs_with_shapes
-                if "admin" not in users[tasks[job["task_id"]]["owner"]["id"]]["groups"]
-                and tasks[job["task_id"]]["target_storage"] is None
-                and tasks[job["task_id"]]["organization"] is None
+                if "admin" not in users[self.tasks[job["task_id"]]["owner"]["id"]]["groups"]
+                and self.tasks[job["task_id"]]["target_storage"] is None
+                and self.tasks[job["task_id"]]["organization"] is None
             )
         )
-        response = self._export_annotations(username, job_id)
-        assert response.data
 
+        self._test_export_annotations(
+            username, job["id"], api_version=api_version, local_download=self.is_local_download(job)
+        )
+
+    @pytest.mark.parametrize("api_version", (1, 2))
```

Comment:
Is it possible to mix these APIs?

Reply:
[Done](https://github.com/cvat-ai/cvat/pull/8216/commits/bf726bbd33112137e7985d4f34cbd973d76b6243)